frontend/public/components/graphs/area.tsx

```diff
@@ -24,6 +28,18 @@ const DEFAULT_SAMPLES = 60;
 const DEFAULT_TICK_COUNT = 3;
 const DEFAULT_TIMESPAN = 60 * 60 * 1000; // 1 hour
 
+export enum AreaChartStatus {
+  error = 'error',
```

Comment:
we usually have enum values in uppercase


```suggestion
  ERROR = 'ERROR',
```